app/models/invoice.rb

```diff
@@ -325,6 +325,8 @@ def should_sync_sales_order?
   def document_invoice_name
     return I18n.t('invoice.prepaid_credit_invoice') if credit?
 
+    return I18n.t('invoice.paid_invoice') if advance_charges?
```

Comment:
Prepaid credit invoice does not include tax, but charge one does.
For some countries mentioning `tax` is mandatory (line 330), so we probably also need it for advance charge invoice. Maybe @mathieu-lago has idea how to display it properly?